src/carts/src/main/java/mushop/carts/repositories/awsdocdb/AwsDocDbDefaultMongoConfiguration.java

```diff
@@ -43,6 +48,7 @@
 /**
  * Overridden default MongoDB configuration class that replaces default codecs by the AWS DocDB specific ones.
  */
+@Requires(env="ec2")
```

Comment:
Better use the constant `io.micronaut.context.env.Environment.AMAZON_EC2`

Reply:
Changed